src/main/java/uk/gov/gchq/palisade/Util.java

```diff
@@ -214,7 +214,8 @@ public static String locateJarFile(final String clazz) {
                 LOGGER.info("Can't determine where {} was loaded from", clazz);
             }
             return (codeSource != null) ? codeSource.getLocation().toString() : null;
-        } catch (Exception e) {
+        } catch (ClassNotFoundException e) {
+            LOGGER.error("LocateJarFile threw an exception and the message is {}", e.getMessage());
```

Comment:
should be more like
`LOGGER.error("LocateJarFile threw an exception", e)`
This maintains whatever pereferences are set for error-logging (whether or not to printstacktrace etc)

Reply:
Applies to all changes in this pr